src/components/header/AuthActions.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import BaseButton from "@/components/base/BaseButton.vue";
+import Hamburger from "@/assets/icons/hamburger.svg?component";
+import { useUserStore } from "@/stores/user";
+const userStore = useUserStore();
+</script>
+
+<template>
+  <div class="actions">
+    <div class="actions__buttons">
+      <BaseButton kind="button" type="secondary" @click="userStore.logout">
```

Comment:
Take into consideration: Since this is a `BaseButton` component probably it's kind will be most often a `button` so maybe it could be set as default value for this prop?

Reply:
kind has default for kind which is `"button"`. I will remove this prop here